crates/pathfinder/src/storage/merkle_tree.rs

```diff
@@ -263,6 +264,7 @@ impl<'a> RcNodeStorage<'a> {
     ///
     /// Does not perform rollback on failure. This implies that you should rollback the [RcNodeStorage's](RcNodeStorage) transaction
     /// if this call returns an error to prevent database corruption.
+    #[allow(unused)]
     fn delete_node(&self, key: StarkHash) -> anyhow::Result<()> {
```

Comment:
If I use these at some point in the future, will it let me know to remove the `#[allow(unused)]`?

Otherwise I would rather mark this as `fn _delete_node` so that I remember to mark it as used again.

Reply:
Hmm good point. The way it seems to me:
- `#[allow(unused)]`: found through autocomplete, use and risk leaving `#[allow(unused)]`
- underscore: not found through prefix, use and rename
- `#[cfg(test)]`: I would assume found through autocomplete, at least `fn delete`, when using must drop `#[cfg(test)]`

Do you still want the underscore?

Reply:
I think its fine as is. Was just curious.

Reply:
Tried the `cfg(test)`, found there's other parts like an `fn _delete` which needed to be un-underscored and marked `cfg(test)` but not too many. Please see https://github.com/eqlabs/pathfinder/pull/117/commits/bc1089c2ff2999bec69932d6dd80ba2e9c24a689 and let me know what you think.